lib/constants.js

```diff
@@ -213,6 +213,8 @@ const PROJECT_COMPONENT_TYPES = {
   COMPONENTS: 'components',
 };
 
+const DEFAULT_PLATFORM_VERSION = '2023.1';
```

Comment:
Do we need to include this on the FE? I imagine that the BE could provide this as a fallback for when we don't pass any value for `platformVersion` in the request. Then we won't run into the issue of devs having outdated versions of the CLI if/when we ever want to change what the value of this default version is.

Reply:
I'll copy Caitlyn on this and see whether she can spin up an endpoint next week.